test/unit/test_read_utils.py

```diff
@@ -193,6 +193,30 @@ def test_mvicuna_empty_input(self):
         )
         self.assertEqual(samtools.count(output_bam), 0)
 
+    def test_cdhit_canned_input(self):
+        samtools = tools.samtools.SamtoolsTool()
+
+        input_bam = os.path.join(util.file.get_test_input_path(self), 'input.bam')
+        expected_bam = os.path.join(util.file.get_test_input_path(self), 'expected.bam')
```

Comment:
Wow, cd-hit produces the exact same output bam as mvicuna? That's creepy... maybe our canned input was simplistic though.

Reply:
Oh this expected bam is not actually used.

Reply:
Oh I see it now.